main.py

```diff
@@ -293,6 +298,12 @@ async def loop(adb_instance: ADB, config: AluneConfig):
         config: An instance of the alune config to use.
     """
     while True:
+        delay_next_game()
+
+        if PAUSE_LOGIC:
+            time.sleep(5)
```

Comment:
Let's stay with asyncio.sleep in async functions :)

Reply:
Hadn't looked very hard and didn't notice, done!